.github/workflows/update_requirements.yml

```diff
@@ -0,0 +1,34 @@
+name: "pylint: linter"
+
+on: 
+  push:
+    branches: main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 'pypy3.9'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pipenv
+        pipenv requirements
+    - name: Commit if update requirements
+      run: |
+        git config user.name bot
+        git config user.email bot@github.com
+
+        if git diff --quiet; then
+          echo "No hay cambios."
+        else
+          git add README.md
```

Comment:
I'm not sure about this. I think this should add requirements.txt ?

Reply:
haha thanks for mentioning this, it's my mistake that I forgot to change what readme.md says to requiremnets.txt, I already made the change.

Also, if everything is fine and you accept the pull request, you will have to give the workflow runner permissions to write to the repository. I leave you a screenshot of how it is done.

go to "settings" in the repository

![image](https://github.com/bypirob/winsi/assets/107817690/ce0a060f-1e83-42e8-8408-98c301321e06)

You go to "settings" in the repository and then to "Actions" being in "general" you go where it says: and mark the same option.

![image](https://github.com/bypirob/winsi/assets/107817690/c4a8f547-0500-4e1a-8310-20ec869c3162)

Reply:
Perfect thanks!